File: test/corpus/languages/lua/fixture/main_test.lua
local user = require("user")

local function test_create_user()
    local u = user.User.new("Alice", "alice@example.com", 30)
    assert(u.name == "Alice")
end

local function test_is_adult()
    local u = user.User.new("Bob", "bob@example.com", 25)
    assert(u:isAdult())
end

return { test_create_user = test_create_user, test_is_adult = test_is_adult }

File: test/corpus/languages/zig/fixture/src/main_test.zig
const std = @import("std");
const user = @import("user.zig");

test "create user" {
    const u = user.User.init("Alice", "alice@example.com", 30);
    try std.testing.expectEqualStrings("Alice", u.name);
}

test "is adult" {
    const adult = user.User.init("Bob", "bob@example.com", 25);
    try std.testing.expect(adult.isAdult());
}
